python/guide/packages/gluon/custom-loss/custom-loss.md

```diff
@@ -1,11 +1,11 @@
 
-# Custom Loss functions in Gluon
+# Custom Loss Blocks
 
-Loss functions are used to train neural networks and to compute the difference between output and target variable. A critical component of training neural networks is the loss function. A loss function is a quantative measure of how bad the predictions of the network are when compared to ground truth labels. Given this score, a network can improve by iteratively updating its weights to minimise this loss. Some tasks use a combination of multiple loss functions, but often you'll just use one. MXNet Gluon provides a number of the most commonly used loss functions, and you'll choose certain functions depending on your network and task. Some common task and loss function pairs include:MXNet's Gluon API provides most commonly used functions. For instance:
+A critical component of training neural networks is the loss function. A loss function is a quantative measure of how bad the predictions of the network are when compared to ground truth labels. Given this score, a network can improve by iteratively updating its weights to minimise this loss. Some tasks use a combination of multiple loss functions, but often you'll just use one. MXNet Gluon provides a number of the most commonly used loss functions, and you'll choose certain functions depending on your network and task. Some common task and loss function pairs include:
```

Comment:
I would reverse this.
Loss functions are crucial/integral to traiing neural networks.

Reply:
Reworded sentence.